### PR TITLE
[6.3] cherry-pick UI get job-invocation ouput with asserts

### PR DESCRIPTION
**Pull request by pondrejk**
Related to  issue #4402
Cherry pick from PR #4430
Testing is currently prevented by https://bugzilla.redhat.com/show_bug.cgi?id=1406315

### CODE REVIEW

**Comment by oshtaier** [comment]
Please rebase and resolve conflicts

File: tests/foreman/ui/test_remoteexecution.py [hunk 1]
```diff
@@ -486,8 +531,18 @@ def test_positive_run_job_template_multiple_hosts(self):
...
+                        status = True
+                    else:
+                        status = False
+                    # get job invocation id from the current url
+                    invocation_id = self.browser.current_url.rsplit('/', 1)[-1]
+                    self.assertTrue(status, 'host output: {0}'.format(
+                        ' '.join(JobInvocation.get_output({
+                             'id': invocation_id,
+                             'host': client.hostname
+                         }))))
```
Comment (oshtaier) [inline tests/foreman/ui/test_remoteexecution.py]: Make unified format for brackets (in every place like here or change there)

**Comment by pondrejk** [comment]
Rebase successful, merging since two acks are in place

**Comment by oshtaier** [comment]
Please, avoid merging by yourself and just ping person directly

**Pull request merged**